api/types/mount.proto

```diff
@@ -42,4 +42,7 @@ message Mount {
 
 	// Options specifies zero or more fstab style mount options.
 	repeated string options = 4;
+	
+	// ExtraOption specifies zero or more extra options except fstab style options
```

Comment:
Could you add example values

Reply:
@AkihiroSuda Ok, different case maybe need different extraoptions, take nydus as an example, nydus-snapshotter(remote-snapshotter) should delivery nydus image metadata file and nydus daemom configs to kata-shim runtime, so these two items can be add to extraoptions to delivery.

Reply:
it is about interface change. If the case is more specifically, like demo or something, it will be easy to understand.

Reply:
@luodw I think it is important to emphasize (in the interface comments) that the `ExtraOptions` field is opaque to containerd and is ignored by the containerd [mount.Mount](https://github.com/containerd/containerd/blob/main/mount/mount.go#L35) API. It aims to provide a way for the snapshotter to pass extra information to the container runtime.

An example of such extra information, in the nydus case, can be the nydus image bootstrap file location and the nydusd configuration, etc. It will be put into [a predefined structure](https://github.com/containerd/nydus-snapshotter/blob/main/snapshot/snapshot.go#L587) in a string format and be decoded in kata runtime, where this information is then passed to nydusd for real usage.

Reply:
@fuweid @AkihiroSuda @ktock Thanks for your comments! Does the approach make sense to you? We don't have an end-to-end demo yet. But we can work on integration first if you prefer to see it working end-to-end wise (which requires modification to nydus-snapshotter, containerd and kata).

Reply:
@fuweid Thanks, I have run a end-to-end demo in my dev machine successfully, and maybe it is more clear to show the code,
1. This [WIP PR](https://github.com/containerd/nydus-snapshotter/pull/36) is the modification for nydus-snapshotter to use the `ExtraOption` and pass config information; 
2. This [WIP PR](https://github.com/kata-containers/kata-containers/pull/4010) is the modification for kata-shim runtime to parse the `ExtraOption` and use it;